TACC/Core-CMS
# Task/gh 171 support css that is forwards compatible

# Overview

1. __Add plugin [`postcss-preset-env`](https://preset-env.cssdb.org/) to support forward-compatible CSS.__
2. Support urgent-need features:
    - [custom media queries](https://preset-env.cssdb.org/features#custom-media-queries)
    - [media query ranges](https://preset-env.cssdb.org/features#media-query-ranges)
3. _Do not remove duplicate CSS i.e. do not sweep bad CSS under the rug._

# Related Issues

- GH-171

# Testing

1. Run `npm ci` _to clean install node dep's_.
1. Run `npm run build` _to build CSS, including a plugin test file_.
1. Open `taccsite_cms/static/site_cms/css/build/_test.css`.
1. Ensure output matches what the files reports the output should be.
